docs/pages/bare/using-expo-client.md

```diff
@@ -53,121 +73,67 @@ if (Constants.appOwnership === 'expo') {
+```
 
-// Get native module or use fallback logger
-const MyAnalytics = NativeModules.MyAnalytics ?? {
-  logEvent: makeLogger('logEvent'),
-  setUser: makeLogger('setUser'),
-};
+This is nice because it ensures the native module _must_ be installed and linked. Two issues with this solution:
 
-// You usually want to wrap native function calls in functions on the JS side to
-// provide TypeScript typing and validation.
+1. You need to know the native module name ahead of time.
+2. You likely want an error to be thrown when a native module is missing in a custom build, this helps you determine if there was a native linking issue.
 
-export function logEvent(options) {
-  MyAnalytics.logEvent(options);
-}
+## Optional imports
 
-export function setUser(options) {
-  MyAnalytics.setUser(options);
-}
-```
-
-### Alternatively, use optional imports
-
-An alternative approach to using `expo-constants` as described above in the [conditional inline requires section](#use-conditional-inline-requires-to-provide-fallbacks) is to use `try/catch` around `require`. There's not any particularly good reason for you to use this in your application code, and it may lead to warning messages when the library throws an error on importing, but it is listed here in case you find yourself with a fitting use case.
+[Metro bundler](/guides/customizing-metro) has built-in support for optional imports. Optional imports refer to wrapping a `require` statement with a `try/catch` to prevent an error from being thrown when the requested module is missing:
 
-The above **MapView.js** would change to look like the following:
+<SnackInline dependencies={['expo-constants', 'react-native-blurhash']}>
 
 ```js
-import * as React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import React from 'react';
+import { View } from 'react-native';
 
-let MapView;
+let Blurhash;
 
 try {
-  const Mapbox = require('@react-native-mapbox-gl/maps').default;
-  Mapbox.setAccessToken('access-token-here');
-  MapView = Mapbox.MapView;
+  Blurhash = require('react-native-blurhash').Blurhash;
 } catch {
-  MapView = props => (
-    <View
-      style={[
-        {
-          backgroundColor: 'lightblue',
-          alignItems: 'center',
-          justifyContent: 'center',
-        },
-        props.style,
-      ]}>
-      <Text>🗺 (Mapbox not available)</Text>
-    </View>
-  );
+  Blurhash = View;
 }
 
-export default MapView;
+export default function App() {
+  return <Blurhash blurhash="LGFFaXYk^6#M@-5c,1J5@[or[Q6." style={{ flex: 1 }} />;
+}
 ```
 
-### **Deprecated**: use the `.expo.[js/json/ts/tsx]` extension to provide Expo Go specific fallbacks
+</SnackInline>
+
+This method is the least reliable because there are a number of reasons that a `require` statement might throw an error. There could be an internal error, the module could be missing, the native module could be linked incorrectly, etc. Ultimately you should try to avoid using this method.
```

Comment:
```suggestion
This method is the least reliable because there are several reasons that a `require` statement might throw an error. For example, there could be an internal error, the module could be missing, the native module could be linked incorrectly, and so on. You should avoid using this method.
```

Let's be authoritative here with the tone for avoiding this method.